chrysom/basicClient.go

```diff
@@ -69,12 +65,6 @@ type BasicClient struct {
 	bucket       string
 }
 
-// Auth contains authorization data for requests to Argus.
-type Auth struct {
-	JWT   acquire.RemoteBearerTokenAcquirerOptions
-	Basic string
-}
```

Comment:
should I add this back in as well?

Reply:
specifically referencing JWT - do I need to move over the RemoteBearerTokenAcquirerOptions from bascule to ancla?

Reply:
@denopink see questions above

Reply:
Good question, let’s chat on a slack huddle 🙂

Reply:
Nah, let's this deleted